### ISSUE TITLE
Migration: Finalised Requisitions show up as `New` Requisitions in OMS

### ISSUE THREAD

**Issue by roxy-dao** [msupply-foundation/open-msupply#5439]
## What went wrong? 😲
Finalised requisitions migrated from Desktop show up as `New` in Open mSupply

![Screenshot 2024-11-18 at 12 40 23 PM](https://github.com/user-attachments/assets/d0179c9e-cc04-4354-9c4b-b168f27fb129)

## How to Reproduce 🔨

Steps to reproduce the behaviour:

1. Finalised a Requisition in Desktop
2. Migrate store to OMS
3. Initialise
4. See that Requisition is now `New`

## Your environment 🌱

- Open mSupply Version: 2.4.0
- Legacy mSupply Central Server Version:

- Platform:

- Database type:

**Comment by roxy-dao** [comment]
Shouldn't be able to edit details of a finalised requisition

**Comment by GeronimoJohn** [comment]
Finalised requisitions are coming in as finalised. Though requisitions in confirmed status are migrating as new, is this expected? 

Desktop: 
<img width="1031" alt="Screenshot 2025-01-09 at 12 26 07 PM" src="https://github.com/user-attachments/assets/f4fe1381-24fe-4025-8e3f-dd7ab7adc146" />

OMS:
![Screenshot 2025-01-09 at 12 26 13 PM](https://github.com/user-attachments/assets/1a8e8ba6-b240-4c33-a977-75ab04e64a5f)

**Comment by roxy-dao** [comment]
> Followed the steps. And found that finalised requisitions in desktop are coming in as finalised. Though requisitions in confirmed status are migrating as new, is this expected?
> Wondering if it might have been fixed?
> 
> @roxy-dao can you see if you can replicate it please 🙏🏽

Yup, CN Requisitions should come as `New`. Are you able to edit the details of a finalised Requisition that has been migrated from OG?

**Comment by GeronimoJohn** [comment]
I can't edit the details of a finalised requisition that's been migrated as well!

https://github.com/user-attachments/assets/80c4e5a1-5d8b-4794-8b08-d669a06bd413

When I hover over the elements and try to click the buttons, nothing happens.

**Comment by roxy-dao** [comment]
Just checked and seems fine to me~? Can reopen if it is an issue. Thanks for checking @GeronimoJohn ~
